integration/integration_test.go

```diff
@@ -35,16 +35,15 @@ func TestSMK6(t *testing.T) {
 		t.Fatalf("sm-k6 binary does not seem to exist, must be compiled before running this test: %v", err)
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	t.Cleanup(cancel)
 
 	outFile := filepath.Join(t.TempDir(), "metrics.txt")
 
 	cmd := exec.CommandContext(ctx, smk6, "run", "-", "-o=sm="+outFile)
 	cmd.Stdin = bytes.NewReader(testScript)
-	err = cmd.Run()
-	if err != nil {
-		t.Fatalf("running sm-k6: %v", err)
+	if k6out, err := cmd.CombinedOutput(); err != nil {
+		t.Fatalf("running sm-k6: %v\n%s", errors.Join(err, ctx.Err()), string(k6out))
```

Comment:
I struggled to reliably get tests to pass without this. I'll probably get rid of these changes in favor of https://github.com/grafana/xk6-sm/pull/73, when and if it gets merged.